x-pack/plugins/observability_ai_assistant/server/service/client/index.ts

```diff
@@ -739,4 +756,18 @@ export class ObservabilityAIAssistantClient {
   deleteKnowledgeBaseEntry = async (id: string) => {
     return this.dependencies.knowledgeBaseService.deleteEntry({ id });
   };
+
+  addResponseLanguage = (messages: Message[], responseLanguage: string): Message[] => {
+    const [systemMessage, ...rest] = messages;
+
+    const extendedSystemMessage: Message = {
+      ...systemMessage,
+      message: {
+        ...systemMessage.message,
+        content: `You MUST respond in the users preferred language which is: ${responseLanguage}. ${systemMessage.message.content}`,
+      },
+    };
+
+    return [extendedSystemMessage].concat(rest);
+  };
```

Comment:
I'm wondering if this should be a class method since it's static. Could just be a utility not attached to the class

Reply:
I get your point, on the other hand, it's only used by this class so having a lose hanging helper function seems like undue decoupling, it's either an extra file or sits below this class in the same file and I don't see the practical difference in that? 

We could make this more clear by setting it to `private` to not dilute the class's interface (the old Java dev in my thought it was private by default).

Or, we could make it more generic, like `extendMessage` but then we'd need to still wrap it in something that pulls the system message and put things together again, not sure if it's worth it.

Reply:
Sure thing.